tests/ui/specs/VirtualList/VirtualGridList/With5-way/VirtualGridList-GT-28455-specs.js

```diff
@@ -0,0 +1,25 @@
+const Page = require('../VirtualGridListPage');
+
+describe('Navigate with 5-way', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should spotlight move `X` button with 5-way [GT-28455]', function () {
+		// Step 4-1: Knobs > VirtualGridList > dataSize > 3
+		// As this View doesn't have 'x' button, Set the data size to 2 so that we can replace it with the top button.
+		Page.inputNumItems.moveTo();
+		Page.spotlightSelect();
+		Page.backSpace();
+		Page.backSpace();
+		Page.backSpace();
+		Page.numPad(2);
```

Comment:
Not 3?
// Step 4-1: Knobs > VirtualGridList > dataSize > 3

Reply:
As this View doesn't have 'x' button, Set the data size to 2 so that we can replace it with the top button.
If set dataSize '3', we cannot replace 'x' button with the top button.

Reply:
Ok. I missed the line 10 description. thanks